notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -194,7 +194,7 @@
         "    \n",
```

Comment:
<p>our -&gt; your</p><p> </p><p>Give -&gt; Specify</p><p> </p><p>predictin -&gt; prediction</p><p> </p><p>input to batch prediction is&nbsp;{"dense_input": ...} -&gt;</p><p>input layer to batch prediction is "dense_input"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>done</p>